ClosedXML/Excel/Drawings/XLPicture.cs

```diff
@@ -140,17 +139,7 @@ public Int32 Height
             }
         }
 
-        public Int32 Id
-        {
-            get { return id; }
-            internal set
-            {
-                if ((_worksheet.Pictures.FirstOrDefault(p => p.Id.Equals(value)) ?? this) != this)
-                    throw new ArgumentException($"The picture ID '{value}' already exists.");
-
-                id = value;
-            }
-        }
+        public Int32 Id { get; internal set; }
```

Comment:
Private set?

Reply:
With `private set`, we won't be able to assign the value read from a file.

Reply:
Sorry, stupid me.